src/test/java/com/github/wovnio/wovnjava/ApiTest.java

```diff
@@ -66,18 +87,32 @@ private String translate(HttpServletRequest request, Settings settings, String h
         return buffer.toByteArray();
     }
 
-    private HttpURLConnection mockHttpURLConnection( int sendLength, int code, String encoding, byte[] response) throws IOException, ProtocolException {
-        InputStream responseStream = new ByteArrayInputStream(response);
+    private HttpURLConnection mockHttpURLConnection(ByteArrayOutputStream requestStream, ByteArrayInputStream responseStream, int code, String encoding) throws IOException, ProtocolException {
         HttpURLConnection mock = EasyMock.createMock(HttpURLConnection.class);
         mock.setDoOutput(true);
         mock.setRequestProperty(EasyMock.anyString(), EasyMock.anyString());
         EasyMock.expectLastCall().atLeastOnce();
         mock.setRequestMethod("POST");
         EasyMock.expect(mock.getResponseCode()).andReturn(code);
         EasyMock.expect(mock.getContentEncoding()).andReturn(encoding);
-        EasyMock.expect(mock.getOutputStream()).andReturn(new ByteArrayOutputStream());
+        EasyMock.expect(mock.getOutputStream()).andReturn(requestStream);
         EasyMock.expect(mock.getInputStream()).andReturn(responseStream);
         EasyMock.replay(mock);
         return mock;
     }
+
+    private static String decompress(byte[] compressed) throws IOException {
+        ByteArrayInputStream bis = new ByteArrayInputStream(compressed);
+        GZIPInputStream gis = new GZIPInputStream(bis);
+        BufferedReader br = new BufferedReader(new InputStreamReader(gis, "UTF-8"));
```

Comment:
Nit: `java.util.Scanner` or `java.util.nio.File` seems better for me, but don't have to change.

Reply:
I don't have a preference, so I'll leave it like this unless I get any further objections.